usaspending_api/common/validator/award_filter.py

```diff
@@ -12,6 +13,16 @@
     "listed on https://api.usaspending.gov/docs/endpoints."
 )
 
+OLD_TAS_FILTER = {
```

Comment:
Usually using "old" means it is around for the interim and will be removed once the old functionality reaches production (or similar scenario)

Is this in the process of being replaced and will be removed later?